src/Core/Verbs/VerbClasses.fs

```diff
@@ -0,0 +1,19 @@
+﻿module VerbClasses
+
+open StringHelper
+open VerbArticle
+open Reflexives
+
+let getClassByThirdPersonSingular = function
+    | form when form |> ends "á"  -> 5
```

Comment:
Do we have names for these classes? Maybe we should use discriminated union instead of plain integers?

Reply:
Well, official grammar addresses them just by [numbers](https://cs.wikipedia.org/wiki/Česká_slovesa#Slovesné_třídy_a_vzory). Although it might still be good idea, e.g. for uniforming serialization and overall approach. However, I would take it separately.

Reply:
Okay, could you create an issue for that?